doc/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/index.md

```diff
@@ -0,0 +1,424 @@
+                            "name":"applicationToken",
+                            "type":[
+                                {
+                                    "type":"string"
+                                },
+                                "null"
+                            ]
+                        },
+                        {
+                            "name":"headerVersion",
+                            "type":[
+                                {
+                                    "type":"int"
+                                },
+                                "null"
+                            ]
+                        },
+                        {
+                            "name":"timestamp",
+                            "type":[
+                                {
+                                    "type":"long"
+                                },
+                                "null"
+                            ]
+                        }
+                    ]
+                },
+                "null"
+            ]
+        },
+        {
+            "name":"recordData",
+            "type":[
+                $                {
+                    record_data_schema
+                },
+                "null"
+            ]
+        }
+    ]
+}
+```
+
+```${record_data_schema}``` - is a variable which is substituted at run time by Kaa HDFS Sink with the Avro schema of the actual logs. 
+This Avro schema is obtained via a [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO) call to Kaa.
+
+## Configuration
+
+The Flume log appender configuration should match to
+[this](https://github.com/kaaproject/kaa/blob/master/server/appenders/flume-appender/src/main/avro/flume-appender-config.avsc) Avro schema.
+
+|name|description|
+|---|---|
+|executorThreadPoolSize|Executor thread pool size|
+|callbackThreadPoolSize|Callback thread pool size|
+|clientsThreadPoolSize|RPC client max thread pool| 
+|includeClientProfile|Client profile data(boolean value)|
+|includeServerProfile|Server profile data(boolean value)|
+|flumeEventFormat|Records container or Generic|
+|hostsBalancing|Prioritized or Round Robin|
+|FlumeNodes|Flume nodes|
+
+<br/>
+
+```json
+{
+    "executorThreadPoolSize":1,
+    "callbackThreadPoolSize":2,
+    "clientsThreadPoolSize":2,
+    "includeClientProfile":{
+        "boolean":true
+    },
+    "includeServerProfile":{
+        "boolean":true
+    },
+    "flumeEventFormat":"RECORDS_CONTAINER",
+    "hostsBalancing":{
+        "org.kaaproject.kaa.server.appenders.flume.config.gen.FlumeNodes":{
+            "flumeNodes":[
+                {
+                    "host":"localhost",
+                    "port":7070
+                },
+                {
+                    "host":"notlocalhost",
+                    "port":7070
+                }
+            ]
+        }
+    }
+}
+```
+
+The following configuration example matches the previous schema.
+
+>**NOTE:**  
```

Comment:
This is a second time the variants for the host balancing is described (the first one is above). Please move this note above (as it has better description).